### PR TITLE
Tuned for rootless operation

### PR DESCRIPTION
**Pull request by rkojedzinszky**
This allows running the image rootless.

### CODE REVIEW

File: Dockerfile
Comment (pierrecdn) [inline Dockerfile:3]: I'll still be the maintainer after the merge ;), so adding this makes no sense.

File: Dockerfile
Comment (pierrecdn) [inline Dockerfile:64]: All of this can make sense, but it actually has nothing to do with rootless.

File: Dockerfile
Comment (pierrecdn) [inline Dockerfile:68]: Why user 33?

File: README.md
Comment (pierrecdn) [inline README.md:27]: Same remark here, doesn't make sense for me to merge this.

File: README.md
Comment (pierrecdn) [inline README.md:99]: Why this reformating?

**Review by pierrecdn — CHANGES_REQUESTED** [review]
Hi,
While I can agree on the intent, this PR does a lot more, and can't be merged as-is.

**Comment by rkojedzinszky** [comment]
Sorry about the huge changes, I am going to close this PR. I will maintain my rootless version.

**Pull request closed without merging**
